www/Makefile

```diff
@@ -47,24 +47,15 @@ template_reference: node_modules/json-schema-ref-parser
 	rm -rf tmp
 
 check-env:
-ifndef AWS_ACCESS_KEY_ID
-	$(error AWS_ACCESS_KEY_ID is undefined)
-endif
 ifndef AWS_BUCKET
 	$(error AWS_BUCKET is undefined)
 endif
 ifndef AWS_DEFAULT_REGION
 	$(error AWS_DEFAULT_REGION is undefined)
 endif
-ifndef AWS_SECRET_ACCESS_KEY
-	$(error AWS_SECRET_ACCESS_KEY is undefined)
-endif
 ifndef FASTLY_API_KEY
 	$(error FASTLY_API_KEY is undefined)
 endif
 ifndef FASTLY_SERVICE_KEY
 	$(error FASTLY_SERVICE_KEY is undefined)
 endif
-ifndef NETLIFYKEY
```

Comment:
Netlify before was explicitly for deploying the site to acceptance. I think we disabled sentinels but have we sound an acceptance solution to replace this?

Reply:
IIRC, it's on @scotthain's roadmap, but right now, deploying from your local machine is the only option.  This change doesn't break that though.  It just means that if all you want to do is deploy the site to prod, you don't need to figure out your netflify key.